Codeinwp/neve
# Development

### Summary


- Restyle product reviews
- Remove `read more` button on grid blog layout
- Fix issue with metabox settings of first post in blog loop
- Remove metabox title and featured image toggles on single product
- Rethink buttons sitewide

### Will affect visual aspect of the product

YES

### Test instructions


- Check out product reviews and single product;
- Check if read more button gets removed - ideally try migrating from old version with settings changed. Nothing should break - wouldn't be bad if another set of eyes looks at this;
- Set the first post from the blog loop metabox settings. This shouldn't affect the blog page - it did before;
- Check if buttons are proper sitewide and that color settings still work as intended.
